### DISCUSSION TITLE
Lesson 5 : brownie networks list incomplete

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#1025] [Q&A] [answered]
Hello 

When i type the command `brownie networks list` my list is less filled thans the video of Patrick collins. 
```
Ethereum
  ├─Mainnet (Infura): mainnet
  ├─Ropsten (Infura): ropsten
  ├─Rinkeby (Infura): rinkeby
  ├─Goerli (Infura): goerli
  └─Kovan (Infura): kovan

Ethereum Classic
  ├─Mainnet: etc
  └─Kotti: kotti

Arbitrum
  └─Mainnet: arbitrum-main
  ....
```
Do you know why ?

**Comment by n4n0b1t3** [comment, marked as answer]
You can setup further networks see [brownie documentation](https://eth-brownie.readthedocs.io/en/latest/network-management.html#adding-a-new-network)

Patrick has a bunch of them.

**Reply by cromewar** [reply]
Yes Patrick has a lot of custom networks I suppose he uses for his daily development workflow.
